package.json

```diff
@@ -30,9 +30,10 @@
   },
   "dependencies": {
     "@manageiq/font-fabulous": "1.0.3",
-    "@manageiq/ui-components": "1.2.17",
+    "@manageiq/ui-components": "1.2.18",
     "@uirouter/angularjs": "1.0.20",
     "actioncable": "5.2.1",
+    "add": "^2.0.6",
```

Comment:
looks like a stray `yarn add yarn add` .. removing in https://github.com/ManageIQ/manageiq-ui-service/pull/1648